frontend/src/components/widgets/Slider/Slider.tsx

```diff
@@ -87,7 +87,20 @@ class Slider extends React.PureComponent<Props, State> {
 
   public componentDidMount(): void {
     // Check default thumb value's alignment vs. slider container
-    this.thumbValueAlignment()
+    for (
+      let i = 0;
+      i < Math.min(this.thumbRef.length, this.thumbValueRef.length);
+      i++
+    ) {
+      this.thumbValueAlignment(
+        this.thumbRef[i].current,
+        this.thumbValueRef[i].current
+      )
+      this.thumbValueAlignment(
+        this.thumbRef[i].current,
+        this.thumbValueRef[i].current
```

Comment:
I don't believe this second call to `this.thumbValueAlignment` is necessary. 
Did some console.log & manual testing to verify that the for loop iterates over both thumb value instances for the range, and removing does not change the alignment behavior.

Reply:
🤦  Yep! you are right. I just removed it. Great catch!